### ISSUE TITLE
Trembling effect with template text adapter

### ISSUE THREAD

**Issue by Scylanth** [amcharts/amcharts5#564]
**Bug description**

Hello,

On Amcharts 5, adding a text adapter on the template of labels can lead to constant repositioning of the labels making a trembling effect.

`
yAxis.get('renderer').labels.template.adapters.add('text', function (text) {
  return text + '%';
});`

Corresponding codepen (WARNING : epilepsy) : [link](https://codepen.io/Scylanth/pen/JjLzyXd)

Could you please have a look and advise ?

Best regards,

**Environment (if applicable)**

amCharts version 5.2.17

**Comment by zeroin** [comment]
We will take a look, but the correct way to format numbers on ValueAxis would be using number formatter. If you need to add % or something else to a number, do it like this:

```
yAxis.set("numberFormatter", am5.NumberFormatter.new(root, {
  "numberFormat": "#,###.00'%'"
});
```

**Comment by martynasma** [comment]
Fixed in [5.2.18](https://github.com/amcharts/amcharts5/releases/tag/5.2.18).

## [5.2.18] - 2022-08-24

### Added
- `disabled` key support added to `DropdownList` item interface.

### Fixed
- Removed leftover debug code in `IndicatorControl`.
- `selected` event in `IndicatorControl` was not firing.
- Adding an adapter on Y-axis' labels could cause flickers on the chart.
- `maxWidth`/`maxHeight` of a `Container` was ignored when arranging objects by Horizontal or Vertical layout.
- `maxWidth`/`maxHeight` of a `Container` was ignored when drawing background.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
